### DISCUSSION TITLE
New problem:Configuration file not found

### DISCUSSION THREAD

**Discussion by ghost** [iliasam/OpenSimpleLidar#14] [Q&A] [answered]
I've run into another problem. My lidar is running fine using the raw data capture exe during the laser alignment step. But when I run LidarScanningTest, the program pops up with “Configuration file not found”. (lidar4_main_code.hex has been loaded.) How can I fix it?

**Comment by iliasam** [comment]
Do "config.ini" file present near LidarScanningTest1.exe ?
https://github.com/iliasam/OpenSimpleLidar/tree/master/PC_utility/LidarScanningTest/LidarScanningTest1/bin/Debug

**Reply by ghost** [reply]
It works now. The current situation is that the light on the pcb board flashes intermittently (corresponding to the coding ring blocking situation), but the computer receives no information (the rxd indicator on the converter barely responds to the hair and occasionally flashes the computer receives a packet.) The radar turns slowly due to the accuracy of my assembly, is this related?

**Comment by iliasam** [comment]
Sorry, I don't understand you. Something wrong with encoder ring? 
You will not get stable data in scanning mode without stably working encoder. 
Rotation must be stable, better to have at least 2 rotations/s.

**Reply by ghost** [reply, marked as answer]
The source of the problem would be the erratic rotation. When I try to manually turn it fast I can get a signal. Looks like I need to reassemble the mechanical part, thanks for your help!
